src/mlpack/methods/ann/layer/lookup.hpp

```diff
@@ -60,15 +65,15 @@ class Lookup
    * forward pass.
    *
    * @param * (input) The propagated input activation.
-   * @param gy The backpropagated error.
-   * @param g The calculated gradient.
+   * @param * (gy) The backpropagated error.
+   * @param * (g) The calculated gradient.
    */
   template<typename eT>
   void Backward(const arma::Mat<eT>& /* input */,
-                const arma::Mat<eT>& gy,
-                arma::Mat<eT>& g);
+                const arma::Mat<eT>& /* gy */,
+                arma::Mat<eT>& /* g */);
```

Comment:
I suggest to revert it back. Doxygen doesn't render it properly.
```suggestion
   * @param gy The backpropagated error.
   * @param g The calculated gradient.
   */
   template<typename eT>
   void Backward(const arma::Mat<eT>& /* input */,
                 const arma::Mat<eT>& gy,
                 arma::Mat<eT>& g);
```

Reply:
But then it will show unused variable warning probably!?

Reply:
No, it doesn't happen with declarations. If you wrote the implementation here then the compiler would write a warning.

Reply:
Ohh... Thanks for the clarification. I will make the change.